locale/es/about/index.md

```diff
@@ -27,7 +27,7 @@ server.listen(port, hostname, () => {
 
 Esto contrasta con el modelo de concurrencia más común de hoy en día, en el que se emplean hilos del Sistema Operativo. Las redes basadas en hilos son relativamente ineficientes y muy difíciles de usar. Además, los usuarios de Node.js están libres de preocuparse por el bloqueo del proceso, ya que no existe. Casi ninguna función en Node.js realiza I/O directamente, por lo que el proceso nunca se bloquea. Por ello, es muy propicio desarrollar sistemas escalables en Node.js.
 
-Si alguno de estos términos no le es familiar, hay un artículo completo en [Blocking vs Non-Blocking](https://github.com/nodejs/node/blob/master/doc/topics/blocking-vs-non-blocking.md).
+Si alguno de estos términos no le es familiar, hay un artículo completo en [Blocking vs Non-Blocking](https://nodejs.org/en/docs/guides/blocking-vs-non-blocking/).
```

Comment:
```suggestion
Si alguno de estos términos no le es familiar, hay un artículo completo en [Blocking vs Non-Blocking](https://nodejs.org/es/docs/guides/blocking-vs-non-blocking/).
```
(The article isn't translated yet, but at least using that URL, the navigation is still in the requested language.)

Reply:
Perfect! I didn't see the language detail, but of course it's better to have the navigation in the same language you're reading the article pointing to it..I totally agree!😉

Reply:
(And if anyone ever does get around to translating it, the link will be pointing to that translation.)